src/OVAL/oval_probe_ext.c

```diff
@@ -781,8 +769,8 @@ int oval_probe_sys_handler(oval_subtype_t type, void *ptr, int act, ...)
 			break;
 		}
 
-		probe_urilen = snprintf(probe_uri, sizeof probe_uri, "%s://%s/%s",
-				OVAL_PROBE_SCHEME, probe_dir, oval_subtype_get_text(type));
+		probe_urilen = snprintf(probe_uri, sizeof probe_uri, "%s://%s",
```

Comment:
Would appreciate feedback on whether this and the similar change on line 835 is the correct change; since `OVAL_PROBE_DIR` is going away, I removed `oval_pext_t->probe_dir` as well. To my knowledge, probe_dir was only used by the OVAL_PROBE_DIR related code, hence removing this seems like a necessary simplification. The following checks on line 775 seems a little extraneous as well, as the `probe_uri` now only consists of `OVAL_PROBE_SCHEME` and the subtype, but I'd like feedback there too.

Reply:
The removal of  `oval_pext_t->probe_dir` looks ok, it seems to be only used for `probe_uri`.
About the checks on 775, yes, the length of `probe_uri` will now be determined by factors under our control, so we can be less worried about it, but there is no ham in keep checking it.